File: 2093-check-if-string-is-a-prefix-of-array/check-if-string-is-a-prefix-of-array.py
class Solution(object):
    def isPrefixString(self, s, words):
        """
        :type s: str
        :type words: List[str]
        :rtype: bool
        """
        strr=""
        for st in words:
            strr+=st
            if strr==s: return True
            if s.find(strr)==-1: return False
        return False

File: 2093-check-if-string-is-a-prefix-of-array/check-if-string-is-a-prefix-of-array.java
class Solution {
    public boolean isPrefixString(String s, String[] words) {
        String str="";
        for(String st:words){
            str+=st;
            if(s.equals(str))
            return true;
            if(s.indexOf(str)==-1)
            return false;
        }
        return false;
    }
}